packages/node-core/src/indexer/project.service.ts

```diff
@@ -115,6 +115,12 @@ export abstract class BaseProjectService<API extends IApi, DS extends BaseDataSo
 
       // Flush any pending operations to set up DB
       await this.storeService.storeCache.flushCache(true, true);
+    } else {
+      assert(startHeight, 'ProjectService must be initalized with a start height in workers');
```

Comment:
<img width="612" alt="image" src="https://github.com/subquery/subql/assets/10431657/377adb59-b6f0-4c51-8735-13627beb13b1">
I was little confused until I found the usage.